bin/make-base-vm

```diff
@@ -208,7 +219,7 @@ WORKDIR /home/$DISTRO
 CMD ["sleep", "infinity"]
 EOF
 
-  docker build --pull -f $OUT.Dockerfile -t $OUT .
+  docker build --network host --pull -f $OUT.Dockerfile -t $OUT .
```

Comment:
unfortunately, this only works on Linux.  if this is desired, it has to be behind a flag or a conditional based on the host OS.

(this also relates to the change above, which points to 127.0.0.1 unconditionally)

Reply:
I will try to investigate the purpose of this, why he had to specify the host network and the relation with apt-cacher. Try to understand why he had to play with `MIRROR_DEFAULT`.

Reply:
For now, if I'm right, it's related to the configuration of apt-cacher when you're using docker.

He's just using adding the following line in `make-base-vm` : https://github.com/devrandom/gitian-builder/blob/32b1145b06f105c2799856d92a788dff0112ce87/bin/make-base-vm#L201

So, I guess if mirrors configurations were working with apt-cacher before, not touching to those configuration when disabling apt-cacher shouldn't be a problem ?

Would keeping `MIRROR_DEFAULT` and docker network like it was be a solution ?